backend/src/main/java/com/redhat/cloud/notifications/auth/rbac/RbacRestClientRequestFilter.java

```diff
@@ -19,7 +18,7 @@ public class RbacRestClientRequestFilter implements ClientRequestFilter {
     public RbacRestClientRequestFilter() {
         String tmp = System.getProperty("develop.exceptional.user.auth.info");
         if (tmp != null && !tmp.isEmpty()) {
-            authInfo = new String(Base64.getEncoder().encode(tmp.getBytes(UTF_8)), UTF_8);
+            authInfo = Base64Utils.encode(tmp);
```

Comment:
Should we remove the if ? I see that the `Base64Utils` already considers nulls

Reply:
An empty String is encoded/decoded to an empty String with Base64 so we don't really need a special condition for that in Base64Utils.

Reply:
I didn't meant that. I was questioning if we need to remove the outer `if` (line 21-20), but current code is fine